src/unitxt/metrics.py

```diff
@@ -511,22 +575,25 @@ def compute(self, references, predictions):
         return super().compute(references, predictions)
 
 
-# Computes char edit distance, ignoring whitespace
-class CharEditDistanceAccuracy(SingleReferenceInstanceMetric):
+# Computes chat edit distance, ignoring whitespace
```

Comment:
chat or char? asking because the class remains char..

Reply:
Fixed.